frontend/src/app/home/components/map/clickables/clickable-offices/clickable-offices.component.ts

```diff
@@ -19,7 +22,17 @@ export class ClickableOfficesComponent implements OnChanges {
       }
     }
   }
+  whatElement(event: Event){
 
+    this.mainService.all.forEach(element => {
+      if (element.numberSVG === event.srcElement.id ) {
+        this.detailArray = element.map();
+        console.log(this.detailArray);
+        this.tooltip = this.detailArray[0].data + " " + this.detailArray[1].data
```

Comment:
Niema sensu mapowanie jeżeli potem i tak musimy po indeksach wybrać to co chcemy, to nigdy nie jest dobrą praktyką.

Lepiej to zrobić tak, jeżeli chcemy mieć tworzenie tooltipa w tym miejscu
``` ts
      if (element.numberSVG === event.srcElement.id) {
        this.tooltip = `${element.firstName} ${element.lastName}`;
      }
```

aczkolwiek ja zrobiłbym dodatkową metodę w modelu, która nam zwroci template do tooltipa.
analogicznie do `map`